js/holmes.js

```diff
@@ -23,7 +23,7 @@
     /* $FlowIssue - doesn't work with umd */
     root.holmes = factory(document);
   }
-})(this, function (document) {
+})(this, (function (document) {
```

Comment:
Could you find the reference about the performance improvement? I can't find the articles about it anymore.